plugin/inc/views/view.class.php

```diff
@@ -873,6 +873,15 @@ protected function displayDataRows($data): void
         }
     }
 
+    private function getFilterButtonOnClickAction(mixed $config, string $buttonName)
+    {
+        if (!is_array($config) || !isset($config[$buttonName]['onClick'])) {
```

Comment:
It is totally safe to just have
```suggestion
        if (!isset($config[$buttonName]['onClick'])) {
```